source/installation-guide/installing-wazuh-agent/solaris/solaris10/wazuh_agent_sources_solaris10.rst

```diff
@@ -25,7 +25,7 @@ Installing Wazuh agent
 
          .. code-block:: console
 
-            # pkgutil -y -i git automake gmake autoconf libtool wget curl gcc5core
+            # /opt/csw/bin/pkgutil -y -i git automake gmake autoconf libtool wget curl gcc5core
```

Comment:
Question: Is the path necessary?

Reply:
It depends. It can be used `export PATH=$PATH:/opt/csw/bin:/usr/sfw/bin` and we could eliminate the entire path.

Reply:
Done :+1: